SPECS/openssl/openssl.spec

```diff
@@ -4,13 +4,17 @@
 Summary:        Utilities from the general purpose cryptography library with TLS implementation
 Name:           openssl
 Version:        1.1.1g
-Release:        9%{?dist}
+Release:        10%{?dist}
 License:        OpenSSL
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Security
 URL:            https://www.openssl.org/
-Source0:        https://www.openssl.org/source/%{name}-%{version}.tar.gz
+Source0:        %{name}-%{version}-hobbled.tar.xz
```

Comment:
It would be good to add a robust comment here, explaining why we're using this source, where we're getting it from and how it differs from the original + the address to the original we're referring to.